daliuge-engine/dlg/drop.py

```diff
@@ -1016,6 +1016,7 @@ def producerFinished(self, uid, drop_state):
             self._finishedProducers.append(drop_state)
             nFinished = len(self._finishedProducers)
             nProd = len(self._producers)
+            self._refCount -= 1
```

Comment:
What is this supposed to mean? `_refCount` is a bit ambigious, could this be changed to `nFinishedProducers` or something similar?